arches_her/templates/views/components/reports/consultation.htm

```diff
@@ -47,7 +44,627 @@ <h1 class="aher-report-toolbar-title"><span class="aher-report-name">{% trans "C
+                                                            <i class="fa fa-trash"></i>
+                                                        </a>
+                                                    </div>
+                                                </td>
+                                            </tr>
+                                        </tbody>
+                                    </table>
+                                </div>
+                            </div>
+                        </div>
+                        <!-- /ko -->
+                    </div>
+                </div>
+                <div class="aher-report-section">
+                    <h2 class="aher-report-section-title"><i data-bind="click: function() {toggleVisibility(visible.systemReferences)}, css: {'fa-angle-double-right': visible.systemReferences(), 'fa-angle-double-up': !visible.systemReferences()}"  class="fa toggle"></i> {% trans "External System References" %}</h2>
+                    <a data-bind="{if: cards['external system references'], click: function(){addNewTile(cards['external system references'])}}" class="aher-report-a"><i class="fa fa-mail-reply"></i> {% trans "Add external system references" %}</a>
+
+                    <!-- Collapsible content -->
+                    <div data-bind="visible: visible.systemReferences" class="aher-report-collapsible-container pad-lft">
+
+                        <!-- ko ifnot: systemReferences() -->
+                        <div class="aher-nodata-note">{% trans "No external system references for this consultation" %}</div>
+                        <!-- /ko -->
+
+                        <!-- ko if: systemReferences() -->
+                        <div class="aher-report-subsection">
+                            <div class="firstchild-container">
+                                <div class="aher-table" data-bind="">
+                                    <table class="table table-striped" cellspacing="0" width="100%">
+                                        <thead>
+                                            <tr>
+                                                <th>{% trans "Reference" %}</th>
+                                                <th>{% trans "Reference Type" %}</th>
+                                                <th>{% trans "Agency" %}</th>
+                                                <th class="aher-table-control all"></th>
+                                            </tr>
+                                        </thead>
+                                        <tbody data-bind="dataTablesForEach: { data: systemReferences, dataTableOptions: createTableConfig(4)}">
+                                            <tr>
+                                                <td data-bind="text: reference"></td>
+                                                <td data-bind="text: referenceType"></td>
+                                                <td data-bind="text: agency"></td>
```

Comment:
agency should be a hyperlink to another resource.

Reply:
Updated